### PR TITLE
Added Tests for OpportunitySideCard

### PR DESCRIPTION
**Pull request by hrahul2605**
PR to issue #931

### CODE REVIEW

**Review by Dishebh — APPROVED** [review]

**Comment by Dishebh** [comment]
@Marvin9 can you pls review this as well?

**Review by Marvin9 — CHANGES_REQUESTED** [review]
Everything is covered except ```handleClick```, Add one more test case under ```<InfoCard />``` and simulate apply button click to assert the status of ```ApplyStatus``` before and after.

[Here](https://github.com/Ignitus/Ignitus-client/blob/4d58c9193dc1c968cda33dc7decc25904baf0314/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultIconInput/index.test.tsx#L42) is the reference to simulate events.

**Comment by hrahul2605** [comment]
Added the test @Marvin9 . Please review it

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  });
+
+  it('should call handleClick once', () => {
+    wrapper = mount(<S.Button {...{ ...ButtonProp }} />);
+    wrapper
+      .find(S.Button)
+      .at(0)
+      .simulate('click');
+    expect(mockClickProp.handleClick).toBeCalledTimes(1);
+  });
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-OpportunitySideCard/Components/index.test.tsx:126]:
```suggestion
  it('should change apply status on Apply button click', () => {
    expect(wrapper.find(S.ApplyStatus).prop('status')).toBe('Apply');
    wrapper
      .find(S.Button)
      .at(0)
      .simulate('click');
    expect(wrapper.find(S.ApplyStatus).prop('status')).toBe('Applied');
  });
```

**Pull request merged**